eurobot_loc/libs/core_functions.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
```

Comment:
Add her cvt_ros_pose2point and point2ros_pose, point2ros_transform ros_transform2point